powerauth-restful-security-spring-annotation/src/main/java/io/getlime/security/powerauth/rest/api/spring/provider/PowerAuthAuthenticationProvider.java

```diff
@@ -150,13 +149,19 @@ private PowerAuthApiAuthenticationImpl validateTokenAuthentication(PowerAuthToke
         soapRequest.setNonce(authentication.getNonce());
         soapRequest.setTimestamp(Long.valueOf(authentication.getTimestamp()));
 
-        final ValidateTokenResponse soapResponse = powerAuthClient.validateToken(soapRequest);
+        final ValidateTokenResponse soapResponse;
```

Comment:
Can we move this inside the `try` block? My OCD does not allow me to merge two different implementations... :D